Test/Unit/Gateway/Response/RefundHandlerTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Alma\MonthlyPayments\Test\Unit\Gateway\Request;
+
+use Alma\API\Entities\Payment as AlmaPayment;
+use Alma\API\Entities\Refund;
+use Alma\MonthlyPayments\Gateway\Response\RefundHandler;
+use Alma\MonthlyPayments\Helpers\Logger;
+use Magento\Payment\Gateway\Data\PaymentDataObject;
+use Magento\Payment\Gateway\Response\HandlerInterface;
+use Magento\Sales\Api\Data\TransactionInterface;
+use Magento\Sales\Model\Order\Payment;
+use Magento\Sales\Model\Order\Payment\Transaction;
+use PHPUnit\Framework\TestCase;
+
+class RefundHandlerTest extends TestCase
+{
+
+    const CREATED_TIMESTAMP = '1654472730';
+
+    public function testRefundHandlerIsInstanceOffHandlerInterface(): void
+    {
+        $this->assertInstanceOf(HandlerInterface::class, $this->createNewRefundHandler());
+    }
+
+    private function createNewRefundHandler(): RefundHandler
+    {
+        return new RefundHandler(...$this->getConstructorDependency());
```

Comment:
if you dont need `getConstructorDependency` and it is not required by an interface, you can remove it from this test class